libs/barista-components/radial-chart/src/radial-chart.ts

```diff
@@ -109,6 +117,25 @@ export class DtRadialChart implements AfterContentInit, OnDestroy {
   }
   private _legendPosition: 'right' | 'bottom' = 'right';
 
+  /** Sets the display mode for the sunburst-chart values to either 'percent' or 'absolute'.  */
+  @Input()
+  set valueDisplayMode(value: 'absolute' | 'percent') {
+    this._valueAsAbsolute = value !== 'percent';
+  }
+  /** @internal Marks if absolute value should be shown or percent instead */
+  _valueAsAbsolute: boolean = true;
```

Comment:
do we need this value, it can be inferred by checking the valueDisplayMode against ` === 'absolute'`.

Reply:
As I remember other comments about removing logic from the template I did this (and also the ngIf you commented), but for me it's ok to remove this var